### PR TITLE
October 2019 Test/Fix

### PR DESCRIPTION
**Pull request by crpietschmann**
This is a test/fix update for the BCDR WDS. Also, the lab was upgraded to VS'19 (Issue #30) and scripts were moved from DropBox into the repo (Issue #31).

The OnPremLinuxVM.zip that contains the Linux VM image (referenced within `/Hands-on lab/NestedDeployments/CustomScripts/HyperVHostConfig.ps1`) was left to be pulled down from the DropBox URL still. The reason for this is the file is over 700MB in size and the Git repo has a file size limit of 100MB, so this needed to remain.

### CODE REVIEW

File: Hands-on lab/Before the HOL - Business continuity and disaster recovery.md [hunk 1]
```diff
@@ -50,15 +50,13 @@ Duration: 20 minutes
...
 
-3.  In the returned search results, select the latest image name. Leave the *Select a Deployment Model* chooser on **Resource Manager**.
+    ![In the Everything blade, Visual Studio Community 2019 (latest release) on Windows Server 2019 (x64) is selected.](images/Setup/image5.png "Marketplace blade")
 
-    ![In the Everything blade, Visual Studio Community 2017 (latest release) on Windows Server 2016 (x64) is selected.](images/Setup/image5.png "Everything blade")
-
-4.  Select **Create**.
+3.  Select **Create**.
 
 5.  Apply the following configuration items on the Basics tab and select **OK**:
```
Comment (emilysaeli) [inline Hands-on lab/Before the HOL - Business continuity and disaster recovery.md]: Where is 4?

File: Hands-on lab/Before the HOL - Business continuity and disaster recovery.md [hunk 2]
```diff
@@ -68,47 +66,47 @@ Duration: 20 minutes
...
     ![Select public inbound ports dropdown with RDP (3389) selected](images/Setup/image27.png "Select public inbound ports dropdown")
 
-9.  Click **Review + create**.
+9.  Select **Review + create**.
 
-     ![Review + create button](images/Setup/image28.png "Review + create button")
+    ![Review + create button](images/Setup/image28.png "Review + create button")
     
-10. After validation, click **Create**. The Azure deployment should begin provisioning. It may take 10+ minutes for the virtual machine provisioning to complete.
+10. After validation, select **Create**. The Azure deployment should begin provisioning. It may take 10+ minutes for the virtual machine provisioning to complete.
```
Comment (emilysaeli) [inline Hands-on lab/Before the HOL - Business continuity and disaster recovery.md]: 10 doesn't show up in the preview. Please check the md

File: Hands-on lab/HOL step-by step - Business continuity and disaster recovery.md [hunk 3]
```diff
@@ -1649,11 +1657,11 @@ In this task you will deploy the website to App Services using Visual Studio, mi
...
 
-62. On the **Add a rule** pane, enter the following values, then select **Add**:
+64. On the **Add a rule** pane, enter the following values, then select **Add**:
 
     - **Name**: BCDRPaaS
-
     - **Backend pool**: BCDRPaaS
+    - **Forwarding protocol**: Match request
 
     ![Add a rule pane.](images/Hands-onlabstep-bystep-Businesscontinuityanddisasterrecoveryimages/media/image308-i.png "Add a rule pane")
```
Comment (emilysaeli) [inline Hands-on lab/HOL step-by step - Business continuity and disaster recovery.md]: 65 is showing up twice in the preview and screws up the rest of the numbering after. Please fix md

File: Hands-on lab/Before the HOL - Business continuity and disaster recovery.md (hunk 1 above)
Comment (crpietschmann) [reply Hands-on lab/Before the HOL - Business continuity and disaster recovery.md]: The numbering has been fixed.

File: Hands-on lab/HOL step-by step - Business continuity and disaster recovery.md (hunk 3 above)
Comment (crpietschmann) [reply Hands-on lab/HOL step-by step - Business continuity and disaster recovery.md]: fixed numbering

**Comment by crpietschmann** [comment]
@emilysaeli  I fixed the review items. Let me know if there's anything else.

**Pull request merged**